src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -2283,13 +2291,21 @@ class ObservationStore(
                 .execute()
           }
 
+          val survivalRate =
+              if (isPermanent && speciesKey.id != null) {
+                totalLiveField.plus(totalLive).mul(100).div(survivalRateDenominator)
```

Comment:
This will do integer math, right? Are we okay with that in terms of rounding behavior? (Same for the similar calculation above.)

Reply:
I think it must be casting to numeric before dividing since `survivalRateDenominator` is numeric, and then after the division casting to integer and doing rounding at that point. I saw the correct rounding happening in the tests so was just assuming that was the case